config/docker/cros-baseline/Dockerfile

```diff
@@ -0,0 +1,44 @@
+FROM debian:bullseye-slim
+MAINTAINER "KernelCI TSC" <kernelci-tsc@groups.io>
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update && apt-get install --no-install-recommends -y \
+    apt-transport-https \
+    ca-certificates \
+    ssh
+
+RUN apt-get update && apt-get install -y \
+    curl \
+    git \
+    inetutils-ping \
+    vim \
+    xz-utils
```

Comment:
Do we really need all this, or even any of this?

Reply:
git yes, to extract private key from ChromiumOS repository.
inetutils yes, we are using to wait DUT to come up.
Rest i will try to remove.

Reply:
That will not work well, as Linux have SYN retransmit interval is arithmetic progression. E.g. first retry X sec, second X*2, X*3.

Reply:
Well, works for me:
```
$ while ! ssh staging.kernelci.org ls /tmp/nope; do date; sleep 1; done
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:27:59 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:04 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:08 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:11 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:15 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:19 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:22 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:26 BST 2022
ls: cannot access '/tmp/nope': No such file or directory
Tue 28 Jun 07:28:29 BST 2022
/tmp/nope
```
(I created the file eventually to stop the loop)

Reply:
In this particular case it looks ok because host is reachable. 
In our case DUT sometimes is unreachable for 10-30 seconds (especially QEMU that is always "first boot").

With exponential syn retry backoff TCP retries look like this (10.255.255.153 host, 1.2.3.4 is imaginary, unreachable DUT)
'''
09:37:37.147578 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052930222 ecr 0,nop,wscale 7], length 0
09:37:38.173547 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052931248 ecr 0,nop,wscale 7], length 0
09:37:40.193530 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052933268 ecr 0,nop,wscale 7], length 0
09:37:44.285616 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052937360 ecr 0,nop,wscale 7], length 0
09:37:52.477535 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052945552 ecr 0,nop,wscale 7], length 0
09:38:08.609560 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052961684 ecr 0,nop,wscale 7], length 0
09:38:41.121525 IP 10.255.255.153.44016 > 1.2.3.4.23: Flags [S], seq 1734573241, win 64240, options [mss 1460,sackOK,TS val 4052994196 ecr 0,nop,wscale 7], length 0
'''
That means if DUT will boot just after 09:38:08, it will have to wait next SYN packet at 09:38:41 (extra up to 33 seconds).
Thats actually reason i added this ping loop in first place, to speed up process.

Reply:
Removed all except ping, as i prefer to discuss it further.
Also ping proven to be useful in identifying and troubleshooting LAVA failures, as it provides extended network information (for example incorrect gateway address and etc).

Reply:
OK, how about ping and then ssh? That way you know the host is up, and then you can wait for the SSH server to be ready.

Reply:
Thats exactly what i am doing now (in other tests, such as tast - too):
https://github.com/kernelci/kernelci-core/pull/1261/commits/bce31bdcd87373bc9d6e866e251c9a1f31559d8f#diff-2bc116dc65df2b517c2e3af68f4b6e7f6307477639f85515be49300a23b4f021R22

Reply:
I see. Then we might add `-o ConnectTimeout` with a reasonable value to ensure the ssh client timeout is long enough (not sure what the default is and that might change anyway). Maybe 5min, so 300 seconds.